tools/dune

```diff
@@ -31,25 +38,11 @@
 
 (ocamllex coqdep_lexer)
 
-(executable
- (name coqwc)
- (public_name coqwc)
- (package coq)
- (modules coqwc)
- (libraries))
-
 (ocamllex coqwc)
 
-(executable
- (name coq_tex)
- (public_name coq_tex)
- (package coq)
- (modules coq_tex)
- (libraries str))
-
-(executable
- (name coq_dune)
- (public_name coq_dune)
+(executables
+ (names coqwc coq_tex coq_dune)
+ (public_names coqwc coq-tex coq_dune)
  (package coq)
- (modules coq_dune)
+ (modules coqwc coq_tex coq_dune)
```

Comment:
Do you actually need to give the module names here (isn't Dune able to figure them out by itself)?